packages/devtools_app/lib/src/timeline/html_frames_bar_plotly.dart

```diff
@@ -174,7 +174,7 @@ class FramesBarPlotly {
           ),
         ),
         marker: Marker(
-          color: colorToCss(mainGpuColor),
+          color: colorToCss(mainRasterColor),
```

Comment:
Since this is in an html file, can you confirm that the timeline still works in the dart:html verison after this change?  That we're not doing something like renaming a string which we also reference (by name) from a css file or something?

Reply:
Tested html app - all is WAI 👍  (after https://github.com/flutter/devtools/pull/1760 lands)